### DISCUSSION TITLE
DoH for Chrome v90

### DISCUSSION THREAD

**Discussion by Ashwin-droid** [AdguardTeam/AdGuardHome#3070] [Q&A] [answered]
With chrome 90 you get doh support which bypasses my Adguard Home
Will turning doh on AGH will block ads on chrome? If yes then continue reading and if no tell me to turn off doh on chrome.

Because of my nature of installation, it runs at a local level see issue #3043 what would be the domain name and how do I get a certificate and how do I install it?

Could you give me step by step instructions for it

**Comment by Ashwin-droid** [comment]
how about blocking chrome's inbuilt doh using AGH which will force chrome to use unsecure dns

**Comment by Ashwin-droid** [comment]
@adguard pls help

**Comment by ainar-g** [comment]
Hello.  This sounds more like a Chrome issue rather than an AGH one.  Have you tried following instructions to change Chrome's DoH server?  For example, [this one], section “How to Setup DNS over HTTPS in Google Chrome?”?

[this one]: https://www.webnots.com/how-to-setup-dns-over-https-in-google-chrome/

**Reply by Ashwin-droid** [reply]
> Hello. This sounds more like a Chrome issue rather than an AGH one. 

But ads are not blocked

**Comment by Ashwin-droid** [comment, marked as answer]
It is locally hosted as a dhcp so only self signed certificate and by the way i got around it by blocking dns queries for doh on chrome
because it is a local edition it is not necessary to use doh instead use your dns upstreams of doh type
anyone facing this issue can just paste these lines in their custom filtering rules

`||dns.google^$important`
`||dns64.dns.google^$important`

**Comment by Ashwin-droid** [comment]
if you find it useful then add it to wiki
